src/verify/VerificationStartScreen.tsx

```diff
@@ -143,25 +137,6 @@ function VerificationStartScreen({
     }
   }, [])
 
-  const canUsePhoneNumber = () => {
-    const countryCallingCode = country?.countryCallingCode || ''
-    if (
-      cachedNumber === phoneNumberInfo.e164Number &&
-      cachedCountryCallingCode === countryCallingCode
-    ) {
-      return true
-    }
-
-    const { SANCTIONED_COUNTRY } = getCountryFeatures(phoneNumberInfo.countryCodeAlpha2)
```

Comment:
this logic is already present in the country picker and i couldn't enter an number from a sanctioned country anyway so i think it is safe to remove